Tools/autotest/rover.py

```diff
@@ -303,6 +303,41 @@ def test_sprayer(self):
         if ex:
             raise ex
 
+    def drive_max_rcin(self, timeout=30):
+        """Test max RC inputs"""
+        self.context_push()
+        ex = None
+
+        try:
+            self.progress("Testing max RC inputs")
+            self.wait_mode("MANUAL")
```

Comment:
```suggestion
            self.change_mode("MANUAL")
```

Reply:
manual is the default as you come into a test IIRC, so it doesn't really matter.

Reply:
Done